setup.py

```diff
@@ -9,6 +9,6 @@
       package_data={'haikufinder': ['cmudict/cmudict.pickle','cmudict/custom.dict',
                                     'data/awkward_breaks','data/awkward_ends','data/awkward_starts',
                                     ]},
-      scripts=['scripts/findhaikus'],
+      scripts=['scripts/findhaikus.py'],
```

Comment:
I think keeping the same name is fine, but the mode of the file has to change back to `0755` so it's executable.